integrationTests/testProcessorNode.go

```diff
@@ -628,9 +628,9 @@ func NewTestProcessorNodeWithCustomDataPool(maxShards uint32, nodeShardId uint32
 	kg := &mock.KeyGenMock{}
 	sk, pk := kg.GeneratePair()
 
-	enabledEpochsConfig := CreateEnableEpochsConfig()
+	enableEpochsConfig := CreateEnableEpochsConfig()
 	genericEpochNotifier := forking.NewGenericEpochNotifier()
```

Comment:
We should use the same epoch notifier for all components, instead of creating a new one for each subcomp

Reply:
removed. These were tentative fixes for integration tests.